libs/langgraph/langgraph/pregel/_loop.py

```diff
@@ -320,6 +324,19 @@ def put_writes(self, task_id: str, writes: WritesT) -> None:
                 w for w in self.checkpoint_pending_writes if w[0] != task_id
             ]
             writes_to_save = writes
+
+        # We never want to persist untracked values in checkpoints
```

Comment:
In most cases, we won't have any untracked values, right? May make sense to only enter this block if we have at least one untracked value?